lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -876,25 +874,22 @@ public function getNowExpression()
     }
 
     /**
-     * Returns a SQL snippet to get a substring inside an SQL statement.
+     * Returns an SQL snippet to get a substring inside the string.
      *
      * Note: Not SQL92, but common functionality.
      *
-     * SQLite only supports the 2 parameter variant of this function.
-     *
-     * @param string   $value  An sql string literal or column name/alias.
-     * @param int      $from   Where to start the substring portion.
-     * @param int|null $length The substring portion length.
-     *
-     * @return string
+     * @param string      $string String to get a substring in, SQL expression.
+     * @param string      $start  Position to start at, SQL expression.
```

Comment:
Similar to the above, `start` being a `string` is confusing to me - we'll need usage examples, IMO

Reply:
There are more than 40 methods in this class which accept SQL expressions. We do need some explanation but it doesn't have to be a phpDoc example for every one of them. Maybe it could be some footnote or a link to the online documentation?

I'd like to leave the doc block in a good enough shape and proceed with the changes. I hope, someone could join later and improve the documentation as well.